editor.planx.uk/src/@planx/components/List/schemas/mocks/GenericUnits.ts

```diff
@@ -89,7 +89,7 @@ export const mockUnitsPayload = {
     "proposal.units.residential.three.identicalUnits": 2,
     "proposal.units.residential.total.listItems": 3,
     "proposal.units.residential.total.units": 5,
-    "proposal.units.residential.total.units.newBuid": 3,
-    "proposal.units.residential.total.units.changeOfUseTo": 2,
+    "proposal.units.residential.total.units.development.newBuild": 3,
+    "proposal.units.residential.total.units.development.changeOfUseTo": 2,
```

Comment:
This now matches the other test example here - 

https://github.com/theopensystemslab/planx-new/blob/be6a176da39bff13cef8164b9b4468f75e0417b3/editor.planx.uk/src/%40planx/components/List/utils.test.ts#L73-L74

`development` is added to the path here - 

https://github.com/theopensystemslab/planx-new/blob/be6a176da39bff13cef8164b9b4468f75e0417b3/editor.planx.uk/src/%40planx/components/List/utils.ts#L68-L72